scripts/clean_test.py

```diff
@@ -118,7 +118,7 @@ def mock_delete_dir(unused_path: str) -> None:
         def mock_delete_file(unused_path: str) -> None:
             check_function_calls['delete_file_is_called'] += 1
         def mock_listdir(unused_path: str) -> List[str]:
-            return ['tmpcompiledjs_dir']
+            return ['some_dir', 'tmpcompiledjs_dir']
```

Comment:
Can you add an assertion to make sure we delete this folder correctly?

Reply:
The mock only lists files in a particular directory but never creates them. So, I think we are safe here. The test only makes sure that the `delete_directory_tree()` method is called for the files beginning with `tmpcompiledjs`.